packages/siop-oid4vp/lib/helpers/Metadata.ts

```diff
@@ -87,26 +87,41 @@ function supportedSubjectSyntaxTypes(rpMethods: string[] | string, opMethods: st
   return supportedSubjectSyntaxTypes
 }
 
+export function collectAlgValues(o: any): string[] {
```

Comment:
Do not name an arg `o` on a public method

Reply:
fixed, also it doesn't need export. (probably a ctrl-alt-m thing)